src/SFA.DAS.Commitments.Api.Client.UnitTests/ApiClientTests/EmployerCommitmentApiTests.cs

```diff
@@ -386,6 +386,16 @@ public void Arrange()
             Assert.Pass();
         }
 
+        [Test]
+        public async Task GetEmployerAccountIds()
+        {
+            var request = new TestRequest(new Uri(ExpectedApiBaseUrl + $"api/employer/ids"), string.Empty);
+            _fakeHandler.AddFakeResponse(request, new HttpResponseMessage { StatusCode = HttpStatusCode.OK, Content = new StringContent(JsonConvert.SerializeObject(new List<long>())) });
+
+            var list = await _employerApiClient.GetAllEmployerAccountIds();
+
+            Assert.Pass();
```

Comment:
This tests that the correct url is used (in FakeResponseHandler), but we could pass a few longs and verify the deserialization by checking the returned list with CollectionAssert.AreEquivalent or similar.